avocado/plugins/list.py

```diff
@@ -132,9 +132,8 @@ def configure(self, parser):
         :param parser: Main test runner parser.
         """
         parser = super(List, self).configure(parser)
-        parser.add_argument('keywords', type=str, default=[], nargs='*',
-                            help="List of paths, aliases or other "
-                            "keywords used to locate tests. "
+        parser.add_argument('reference', type=str, default=[], nargs='*',
+                            help="List of test references (aliases or paths). "
```

Comment:
Not really related to this PR, but I don't see a problem with it... (maybe the first 2 commits could be even cherry-picked...)